lib/urbanopt/geojson/zoning.rb

```diff
@@ -74,33 +75,38 @@ def self.divide_floor_print(floor_print, perimeter_depth, runner)
           new_point = vertices[i] + vector
           new_vertices << new_point
         end
-        normal = OpenStudio.getOutwardNormal(new_vertices)
-        if normal.empty? || normal.get.z < 0
-          runner.registerWarning('Wrong direction for resulting normal, will not divide')
-          return [floor_print]
-        end
-        self_intersects = OpenStudio.selfIntersects(OpenStudio.reverse(new_vertices), 0.01)
-        if OpenStudio::VersionString.new(OpenStudio.openStudioVersion) < OpenStudio::VersionString.new('1.12.4')
-          self_intersects = !self_intersects
-        end
-        if self_intersects
-          runner.registerWarning('Self intersecting surface result, will not divide')
-        end
-        result << t_inv * new_vertices
-        (0...n).each do |i|
-          perim_vertices = OpenStudio::Point3dVector.new
-          if i == (n - 1)
-            perim_vertices << vertices[i]
-            perim_vertices << vertices[0]
-            perim_vertices << new_vertices[0]
-            perim_vertices << new_vertices[i]
-          else
-            perim_vertices << vertices[i]
-            perim_vertices << vertices[i + 1]
-            perim_vertices << new_vertices[i + 1]
-            perim_vertices << new_vertices[i]
+        unless scale = true
```

Comment:
this should be == true.  Also how are you testing this to see that it doesn't go in the else clause? The tests are all passing so I'm not sure how to test it...

Reply:
when making this change, I get 4 test failures...maybe that will help narrow down the issue @tanushree04 ?

Reply:
@kflemin I updated to ==true and rewrote the logic its not failing for the else condition now